Update key-vault-certs.md (#86115)
Azure/azure-content: articles/application-gateway/key-vault-certs.md

Set-AzApplicationGateway -ApplicationGateway $appgw
```

> Note: If it is required that Application Gateway syncs the last version of certificate with the KeyVault, please provide the version-less secretId (no hash), by replacing in the example above the line 
> ```
> $secretId = $secret.Id # https://<keyvaultname>.vault.azure.net/secrets/<hash>
> ```
> with
> ```
> $secretId = $secret.Id.Replace($secret.Version, "") # https://<keyvaultname>.vault.azure.net/secrets/
> ```
> 

Reviewer: ```suggestion
> [!Note]
> If you require Application Gateway to sync the last version of the certificate with the KeyVault, provide the version-less secretId (no hash).  To do so, replace the following line in the previous example:
>
> ```
> $secretId = $secret.Id # https://<keyvaultname>.vault.azure.net/secrets/<hash>
> ```
> with
>
> ```
> $secretId = $secret.Id.Replace($secret.Version, "") # https://<keyvaultname>.vault.azure.net/secrets/
> ```

```

Reviewer: @sevelho Reword the note and attempt to put the note in proper markdown format. 

Manually formatting markdown notes is not easy when you don't get a build to check it. If you create PRs in the private repo (azure-docs-pr), you'll get preview builds and various other helpful checks so changes are easier to merge.
